doc/decisions/0_drafts/transformations.md

```diff
@@ -0,0 +1,152 @@
+# Transformations
+
+## Problem
+
+Certain plugins, e.g. `rename`, perform extensive transformations on keys and keysets within Elektra.
+Those transformations include, but are not limited to:
+
+- Changing the names of the keys inbetween `kdbGet` and `kdbSet` operations
```

Comment:
Wouldn't this "inbetween" also be applicable to the others (value and metadata)?

```suggestion
- Changing the names of the keys back and forth
```